benchmarks/java/pom.xml

```diff
@@ -14,7 +14,10 @@
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
-      <version>3.5.0</version>
+	  <version>1.0</version>
```

Comment:
This seems wrong?

Reply:
Then what version number should I set?

Reply:
I think the benchmark should use protobuf-java in the same repo not just for a specific version?

Reply:
In that case, can you use something like "<version>${protobuf.version}</version>" here and pass "-Dprotobuf.version=..." when invoking mvn in the Makefile?

Reply:
Done